tests/test_query_engine.py

```diff
@@ -389,6 +389,48 @@ class Cohort:
     assert extract(Cohort, MockBackend, database) == expected
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "filtered_table,expected",
+    [
+        (
+            table("clinical_events").filter(
+                "result", greater_than=15, include_null=True
```

Comment:
I think this is fine for now, but longer term I think we want some way of composing arbitrary filter expressions using something like Django's [Q object](https://docs.djangoproject.com/en/3.2/topics/db/queries/#complex-lookups-with-q-objects). We could still present an `include_null` option to the user if we think that's a common enough need to deserve its own option. But under the hood it would be better if the AST could represent this as an OR over two filter expressions as that keeps things at a higher level of generality.